src/modules/glaxnimate/CMakeLists.txt

```
@@ -171,6 +171,7 @@ function(mlt_add_glaxnimate_module ARG_TARGET)
       ${GLAX_SOURCES}
     )
 
+    generate_export_header(mltglaxnimate)
```

```suggestion
    generate_export_header(${ARG_TARGET})
```

this means we need to do #ifdef in glaxnimate_producer.cpp probably best with the Qt version as condition